src/main/kotlin/study/racingcar/step2/Calculator.kt

```diff
@@ -0,0 +1,58 @@
+package study.racingcar.step2
+
+import study.racingcar.step2.operation.AbstractCalculation
+import study.racingcar.step2.operation.DivisionCalculation
+import study.racingcar.step2.operation.MinusCalculation
+import study.racingcar.step2.operation.MultiplicationCalculation
+import study.racingcar.step2.operation.PlusCalculation
+import java.util.HashMap
+
+class Calculator {
+    private val COMMON_SPACE = " "
+    private val calculationMap: MutableMap<String, AbstractCalculation>
+    private var expression: String? = null
+    private lateinit var expressionValues: Array<String>
```

Comment:
`Array`를 사용하신 이유가 있을까요?

`List` 타입을 활용해보세요.

```suggestion
    private lateinit var expressionValues: List<String>
```

Reply:
추가로 이 값이 인스턴스 변수로 정의될 필요가 있을지에 대해서도 고민해보시면 좋을 것 같습니다.